src/api/shopper-insights/component.test.js

```diff
@@ -92,35 +157,39 @@ describe("shopper insights component - getRecommendedPaymentMethods()", () => {
           email: "email@test.com",
           phone: {
             countryCode: "1",
-            nationalNumber: "2345678901",
+            nationalNumber: "2345678905",
           },
         },
       })
-    ).rejects.toThrow("This is an API error");
-    expect(callRestAPI).toHaveBeenCalled();
+    ).rejects.toThrow(
+      new Error(
+        `https://api.paypal.com/v2/payments/find-eligible-methods returned status 400\n\n{"name":"ERROR","message":"This is an API error"}`
+      )
+    );
+    expect(request).toHaveBeenCalled();
     expect.assertions(2);
   });
 
   test("create customer payload with email and phone number", async () => {
     const shopperInsightsComponent = getShopperInsightsComponent();
     await shopperInsightsComponent.getRecommendedPaymentMethods({
       customer: {
-        email: "email@test.com",
+        email: "email10@test.com",
         phone: {
           countryCode: "1",
-          nationalNumber: "2345678901",
+          nationalNumber: "2345678906",
         },
       },
     });
 
-    expect(callRestAPI).toHaveBeenCalledWith(
+    expect(request).toHaveBeenCalledWith(
```

Comment:
is there a reason you chose to expect `request` to be called instead of expecting `callMemoizedRestAPI` to be called? just curious (the change from `data` to `json` prompted this question for me)